source/excerptexport/views.py

```diff
@@ -65,6 +75,35 @@ def create_excerpt_export(request):
     return render(request, 'excerptexport/templates/create_excerpt_export.html', viewContext)
 
 
+@login_required(login_url='/admin/')
+def show_downloads(request):
+    view_context = {}
+
+    files = OutputFile.objects.filter(extraction_order__orderer=request.user, extraction_order__state=ExtractionOrderState.FINISHED)
+    view_context['files'] = files
+    return render(request, 'excerptexport/templates/show_downloads.html', view_context)
+
+
+def download_file(request):
+
+    file_id = int(request.GET['file'])
+    output_file = get_object_or_404(OutputFile, public_identifier=file_id, deleted_on_filesystem=False)
+    if not output_file.file:
+        return HttpResponseNotFound('<p>No output file attached to output file record.</p>')
+
+    download_file_name = settings.APPLICATION_SETTINGS['download_file_name'] % {'id': output_file.public_identifier, 'name': os.path.basename(output_file.file.name)}
+    absolute_file_path = os.path.abspath(settings.APPLICATION_SETTINGS['data_directory'] + '/' + os.path.basename(output_file.file.name))
```

Comment:
`settings.APPLICATION_SETTINGS['data_directory']` is already absolute. Give reason why it has to be processed by `abspath` again, or just use it as-is.